bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/Polish.java

```diff
@@ -189,8 +189,8 @@ public Polish() {
         yml.addDefault(Messages.FORMATTING_GENERATOR_TIER2, "II");
         yml.addDefault(Messages.FORMATTING_GENERATOR_TIER3, "III");
         yml.addDefault(Messages.FORMAT_PAPI_PLAYER_TEAM_TEAM, "{TeamColor}[{TeamName}]");
-        yml.addDefault(Messages.FORMAT_PAPI_PLAYER_TEAM_SHOUT, "&6[SHOUT]");
-        yml.addDefault(Messages.FORMAT_PAPI_PLAYER_TEAM_SPECTATOR, "&7[SPECTATOR]");
+        yml.addDefault(Messages.FORMAT_PAPI_PLAYER_TEAM_SHOUT, "&6[OGÓLNY]");
```

Comment:
lepsze byłoby chyba "GLOBALNY" zamiast "OGÓLNY"

Reply:
to zależy od regionu, ja jestem ze wschodu polski i tak się u mnie mówi, ale okej zmieniłem